lib/ansible/plugins/connection/ssh.py

```diff
@@ -686,15 +686,15 @@ def _send_initial_data(self, fh, in_data, ssh_process):
         try:
             fh.write(to_bytes(in_data))
             fh.close()
-        except (OSError, IOError):
+        except (OSError, IOError) as e:
             # The ssh connection may have already terminated at this point, with a more useful error
             # Only raise AnsibleConnectionFailure if the ssh process is still alive
             time.sleep(0.001)
             ssh_process.poll()
             if getattr(ssh_process, 'returncode', None) is None:
                 raise AnsibleConnectionFailure(
-                    'SSH Error: data could not be sent to remote host "%s". Make sure this host can be reached '
-                    'over ssh' % self.host
+                    'Data could not be sent to remote host "%s". Make sure this host can be reached '
```

Comment:
Why remove the `SSH Error:` prefix here, but not below? You should probably just leave it here instead of removing it

Reply:
we already add an [ERROR] and 'connection name' ... seemed redundant, but i missed the other one